### PR TITLE
build: add configuration for kokoro publish

### PR DESCRIPTION
**Pull request by bcoe**

### CODE REVIEW

File: .trampolinerc [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Google LLC
```
Comment (jackwotherspoon) [inline .trampolinerc:1]:
```suggestion
# Copyright 2023 Google LLC
```

File: .kokoro/trampoline_v2.sh [hunk 2]
```diff
@@ -0,0 +1,490 @@
+#!/usr/bin/env bash
+# Copyright 2020 Google LLC
```
Comment (jackwotherspoon) [inline .kokoro/trampoline_v2.sh:2]:
```suggestion
# Copyright 2023 Google LLC
```

File: .kokoro/populate-secrets.sh [hunk 3]
```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+# Copyright 2020 Google LLC.
```
Comment (jackwotherspoon) [inline .kokoro/populate-secrets.sh:2]:
```suggestion
# Copyright 2023 Google LLC.
```

File: .kokoro/publish.sh [hunk 4]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Copyright 2018 Google LLC
```
Comment (jackwotherspoon) [inline .kokoro/publish.sh:3]:
```suggestion
# Copyright 2023 Google LLC
```

**Review by jackwotherspoon — COMMENTED** [review]
Should the license headers be updated to 2023?

**Review by jackwotherspoon — APPROVED** [review]

**Review by ruyadorno — APPROVED** [review]

**Pull request merged**